### PR TITLE
Feature/4638 array right rotation

### PR DESCRIPTION
**Pull request by SarthakChaudhary46**
Done with the changes✅

Done with the requested changes.✅

### CODE REVIEW

**Review by BamaCharanChhandogi — CHANGES_REQUESTED** [review]
please change these.

File: src/test/java/com/thealgorithms/others/ArrayRightRotation.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.thealgorithms.others;
+
+class ArrayRightRotation {
```
Comment (BamaCharanChhandogi) [inline src/test/java/com/thealgorithms/others/ArrayRightRotation.java:3]: add `public` before class

File: src/test/java/com/thealgorithms/others/ArrayRightRotation.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    static int[] rotateRight(int[] arr, int k) {
```
Comment (BamaCharanChhandogi) [inline src/test/java/com/thealgorithms/others/ArrayRightRotation.java:4]: same add `public` keyword

**Comment by SarthakChaudhary46** [comment]
> please change these.

- done with all the changes✅

File: src/test/java/com/thealgorithms/others/ArrayRightRotationTest.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.thealgorithms.others;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ArrayRightRotationTest {
+
+    @Test
```
Comment (BamaCharanChhandogi) [inline src/test/java/com/thealgorithms/others/ArrayRightRotationTest.java:9]: add different test cases. You can reference this code.
[https://github.com/TheAlgorithms/Java/blob/master/src/test/java/com/thealgorithms/others/ArrayLeftRotationTest.java](url)

**Comment by SarthakChaudhary46** [comment]
Please have a look now!

**Comment by BamaCharanChhandogi** [comment]
thanks, LGTM.

**Review by BamaCharanChhandogi — APPROVED** [review]

**Pull request merged**